commands/delete_cluster.go

```diff
@@ -43,58 +69,104 @@ func init() {
 	DeleteCommand.AddCommand(DeleteClusterCommand)
 }
 
-// checks preconditions
-func checkDeleteCluster(cmd *cobra.Command, args []string) error {
-	if cmdClusterID == "" {
-		return errors.New(color.RedString("Please select a cluster to delete"))
-	}
+// validationOutput runs our pre-checks.
```

Comment:
Nitpick: it's not exported but I'd like it if this was `deleteClusterValidationOutput` to match GoDoc convention